components/infieldbutton/stories/infieldbutton.stories.js

```diff
@@ -4,79 +4,94 @@ import { Template } from "./template";
-	description: "The Infield button component is...",
-	component: "Infieldbutton",
-	argTypes: {
-		size: {
-			name: "Size",
-			type: { name: "string", required: true },
-			table: {
-				type: { summary: "string" },
-				category: "Component",
-			},
-			options: ["s", "m", "l", "xl"],
-			control: "select",
-		},
-		quiet: {
-			name: "Quiet",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "Component",
-			},
-			control: "boolean",
-		},
-		position: {
-			name: "Position",
-			type: { name: "string", required: true },
-			table: {
-				type: { summary: "string" },
-				category: "Component",
-			},
-			options: ["left", "right"],
-			control: "select",
-		},
-		iconName: {
-			...(IconStories?.argTypes?.iconName ?? {}),
-			if: false,
-		},
-		isDisabled: {
-			name: "Disabled",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "State",
-			},
-			control: "boolean",
-		},
-		isInvalid: {
-			name: "Invalid",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "State",
-			},
-			control: "boolean",
-		},
-	},
-	args: {
-		rootClass: "spectrum-InfieldButton",
-		size: "m",
-		position: "left",
-		iconName: "Add",
-	},
-	parameters: {
-		actions: {
-			handles: [],
-		},
-		status: {
-			type: process.env.MIGRATED_PACKAGES.includes("infieldbutton")
-				? "migrated"
-				: undefined,
-		},
-	},
+  title: "Components/In-field button",
+  description: "The In-field button component is a button used inside a textfield",
+  component: "Infieldbutton",
+  argTypes: {
+    size: {
+      name: "Size",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["s", "m", "l", "xl"],
+      control: "select"
+    },
+    isQuiet: {
+      name: "Quiet",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean"
+    },
+    position: {
+      name: "Position",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["left", "right"],
+      control: "select"
+    },
+    iconName: {
+      ...IconStories?.argTypes?.iconName ?? {},
+      if: false,
+    },
+    isDisabled: {
+      name: "Disabled",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean"
+    },
+    isInvalid: {
+      name: "Invalid",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean"
+    },
+  },
+  args: {
+    rootClass: "spectrum-InfieldButton",
+    size: "m",
+    position: "left",
+    iconName: "Add",
+    isQuiet: false
```

Comment:
Noticed Disabled does not have a default and initially displays with `Set Boolean` do we want it to match isQueit and start off as `false`